ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml

```diff
@@ -0,0 +1,49 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v24
+images:
+- dockerfile_path: Dockerfile.ci.test
+  to: console-plugin-template
+promotion:
+  name: "4.12"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-plugin-template
```

Comment:
We should have a test that runs the linter on the repo, but that can be a follow up.